docs/source/basic_examples.rst

```diff
@@ -1022,6 +1022,39 @@ useful key for metal-containing molecules. You can use the
     stk.MoleculeMongoDb = _old_molecule_init
     pymongo.MongoClient = _mongo_client
 
+Accessing Vertex Classes
```

Comment:
I would probably rename this something like "Creating New Topology Graphs with Existing Vertices" or sometrhing better if you like